docs/docs/concepts/Sync_Async.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Sync & Async"
+linkTitle: "Sync & Async"
+weight: 3
+description: >
+    Sync and Async Functions
+---
+
+By default, each time a function is invoked, it is invoke synchronously. This means that the atomicity property is promised (no other commands will be invoke or Redis while RedisGears function is running). Atomicity property has some great advantages:
+
+* You can update multiple keys at once and be sure any other client will see the entire update (and not partial updates).
+* You can be sure the data in Redis are not changed while processing it.
+
+On major disadvantage of the atomicity property is that during the entire invocation Redis is blocked and can not serve any other clients.
+
+Triggers and Functions attempt to give a better flexibility to the function writer and allow to invoke function on the background. When function is invoke on the background it can not touch the Redis key space, To touch the Redis key space from the background, the function must block Redis and enter an atomic section where the atomicity property is once again guaranteed.
```

Comment:
```suggestion
By default, each time a function is invoked, it is executed synchronously. This ensures the atomicity property, meaning that no other commands will be executed on Redis while the function is running. The atomicity property offers several advantages:

* Multiple keys can be updated simultaneously, guaranteeing that other clients see the complete update rather than partial updates.
* The data in Redis remains unchanged while it is being processed.

However, the major disadvantage of the atomicity property is that Redis is blocked throughout the entire invocation, preventing it from serving other clients.

Triggers and Functions aim to provide greater flexibility to function writers by enabling the invocation of functions in the background. When a function is invoked in the background, it cannot directly access the Redis key space. To interact with the Redis key space from the background, the function must block Redis and enter an atomic section where the atomicity property is once again guaranteed.
```